datacenterlight/tests.py

```diff
@@ -122,10 +122,11 @@ def test_create_vm_task(self):
             msg = subscription_result.get('error')
             raise Exception("Creating subscription failed: {}".format(msg))
 
+        referer_url = 'test/referer/url'
         async_task = create_vm_task.delay(
```

Comment:
Add a check to verify that referer_url has been saved correctly, may be after the VM has been created ?

Reply:
added a test to check for the referer_url in the same test method. But in a separate PR, we can move it into a separate test, testing for all the attributes post-save.